### ISSUE TITLE
encryption fails on Nethunter 3 with Nexus 7 2013 wifi lolipop

### ISSUE THREAD

**Issue by thesle3p** [offensive-security/kali-nethunter#343]
the suggested steps of 
adb shell
su
mv /data/local/userinit.sh /data/local/userinit_old.sh

 do not work on Nethunter 3 with lolipop as it can not find userinit.sh in /data/local/ and encrypting the device any way (Worth a try right?) results in a device reset.

**Comment by jmingov** [comment]
Hello @thesle3p,

`mv /data/local/userinit.sh /data/local/userinit_old.sh` is no longer used in 3.0

Atm, the easiest way is to remove the chroot from nethunter app, encrypt the device and then reinstall the chroot. (I dindt tested this, but if the problem is that nethunter chroot is mounted, it should work) 

I will add to the app a way to disable the Kali mountAtBoot, so we can encrypt the device without removing the chroot.

Jaime

**Comment by binkybear** [comment]
This seems to be an issue with all older devices which don't encrypt by default in marshmallow.  Newer devices like the Nexus 6 are encrypted by default and don't have any issues.  I tried encrypted the Nexus 5 and it would bootloop trying to mount /data.

The fixes that @jmingov tried do not fix encryption issue but are still useful.  However, I did find a workaround to getting the Nexus 5 encrypted:

* OEM Unlock
* Flash factory
* Load into device
* Encrypt and reboot
* Flash TWRP/SuperSU/Nethunter

It seems like the order is most important.  Like I mentioned, since the Nexus 6 already comes encrypted it never had any issues with NH.  If you try to encrypt AFTER NH is installed you will bootloop.

Some forums suggest this is a SuperSU error.  Going forward this will probably not be an issue on newer devices because they will be encrypted by default.  Removing Nethunter or init.d script do not seem to fix the issue either so I'm leaning toward this being a SuperSU issue.
